critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -70,6 +73,31 @@
         return confirmationMessage;                                // Gecko and WebKit
       });
 
+      oldReviewLength = reviewContentEditor.value().length
+      languageSelector = $('#review-language')[0]
+      // Detect the language of the review on change
+      // codemirror is the underlying editor behind SimpleMDE.
+      reviewContentEditor.codemirror.on("change", function() {
+        const MIN_DIFFERENCE = 10
+
+        var currentReviewContent = reviewContentEditor.value()
+        var currentReviewLength = currentReviewContent.length
+        if(currentReviewLength == 0) { return; }
+
+        // Don't call too often.
+        var difference = Math.abs(currentReviewLength - oldReviewLength)
```

Comment:
I love the use of abs function here. Takes the case pf deletion of text too. Good job !